app/assets/javascripts/index.js

```diff
@@ -238,11 +238,37 @@ $(document).ready(function () {
     L.marker(center, {icon: OSM.getUserIcon()}).addTo(map);
   });
 
+  var isMobile = {
+    Windows: function() {
+        return /IEMobile/i.test(navigator.userAgent);
+    },
+    Android: function() {
+        return /Android/i.test(navigator.userAgent);
+    },
+    BlackBerry: function() {
+        return /BlackBerry/i.test(navigator.userAgent);
+    },
+    iOS: function() {
+        return /iPhone|iPad|iPod/i.test(navigator.userAgent);
+    },
+    any: function() {
+        return (isMobile.Android() || isMobile.BlackBerry() || isMobile.iOS() || isMobile.Windows());
+    }
+  };
+
+  function url() {
```

Comment:
Typically you wouldn't want to use a function for this or below just for one variant, I just wanted to leave the door open for potential support of at least GoMap!!. But  @bryceco needs to way in on that.